### ISSUE TITLE
Crash with Create

### ISSUE THREAD

**Issue by WastelandOasis** [Crendgrim/DynamicCrosshairCompat#16]
Getting a crash to desktop when I look at a blaze spawner with the blaze burner block in hand from create.

[00:09:56] [Render thread/ERROR]: Unreported exception thrown!
java.lang.ClassCastException: class net.minecraft.class_2636 cannot be cast to class io.github.fabricators_of_create.porting_lib.mixin.common.accessor.BaseSpawnerAccessor (net.minecraft.class_2636 is in unnamed module of loader net.fabricmc.loader.impl.launch.knot.KnotClassLoader @4d49af10)
	at mod.crend.dynamiccrosshair.compat.create.CreateItemHandler.computeFromItem(CreateItemHandler.java:215) ~[dynamiccrosshair-compat-3.2.1+1.19.2.jar:?]
	at mod.crend.dynamiccrosshair.compat.create.ApiImplCreate.computeFromItem(ApiImplCreate.java:52) ~[dynamiccrosshair-compat-3.2.1+1.19.2.jar:?]
	at mod.crend.dynamiccrosshair.component.CrosshairHandler.buildCrosshairAdvancedFromItem(CrosshairHandler.java:60) ~[dynamiccrosshair-5.0+1.19.2-fabric.jar:?]
	at mod.crend.dynamiccrosshair.component.CrosshairHandler.buildCrosshairAdvancedByHand(CrosshairHandler.java:88) ~[dynamiccrosshair-5.0+1.19.2-fabric.jar:?]
	at mod.crend.dynamiccrosshair.component.CrosshairHandler.buildCrosshairAdvanced(CrosshairHandler.java:93) ~[dynamiccrosshair-5.0+1.19.2-fabric.jar:?]
	at mod.crend.dynamiccrosshair.component.CrosshairHandler.buildCrosshairDynamic(CrosshairHandler.java:168) ~[dynamiccrosshair-5.0+1.19.2-fabric.jar:?]
	at mod.crend.dynamiccrosshair.component.CrosshairHandler.buildCrosshair(CrosshairHandler.java:222) ~[dynamiccrosshair-5.0+1.19.2-fabric.jar:?]
	at mod.crend.dynamiccrosshair.component.CrosshairHandler.checkShowCrosshair(CrosshairHandler.java:262) ~[dynamiccrosshair-5.0+1.19.2-fabric.jar:?]
	at mod.crend.dynamiccrosshair.component.CrosshairHandler.tick(CrosshairHandler.java:283) ~[dynamiccrosshair-5.0+1.19.2-fabric.jar:?]
	at net.minecraft.class_329.handler$blf000$dynamiccrosshair$tickDynamicCrosshair(class_329.java:11542) ~[client-intermediary.jar:?]
	at net.minecraft.class_329.method_1748(class_329.java:1305) ~[client-intermediary.jar:?]
	at net.minecraft.class_329.method_39191(class_329.java:1271) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1574(class_310.java:1792) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1523(class_310.java:1135) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.method_1514(class_310.java:768) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.method_44604(Main.java:244) [client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:51) [client-intermediary.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:462) [fabric-loader-0.14.18.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) [fabric-loader-0.14.18.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) [fabric-loader-0.14.18.jar:?]

**Comment by Crendgrim** [comment]
What version of Create are you using?

**Comment by Crendgrim** [comment]
Ah, I was able to replicate it. Will be fixed with the next release of DCC.

**Comment by WastelandOasis** [comment]
Sorry about that. Glad you were able to figure it out!

**Comment by Crendgrim** [comment]
I just pushed DynamicCrosshairCompat 3.3 which should fix this crash by not trying (and failing) to be as clever; the disadvantage being that now it will appear as though blaze burners can be used on any monster spawner. It is surprisingly difficult to figure out whether a spawner can spawn a specific mob.

I also pushed DynamicCrosshair 5.3 which will silence any such crashes in the future (write them out to the game log, but recover and just keep going).
